spec/services/visit_timeline_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe VisitTimeline do
+  let(:instance) { described_class.new(visit) }
+  let(:visit) { FactoryGirl.create(:visit) }
+
+  describe '#events' do
+    subject(:events) { instance.events }
```

Comment:
In my view `subject` doesn't have to be exclusively the instance of the objects, it can also be the method being described. In this case what's under test is the collection of events. I somewhat follow http://betterspecs.org/#subject with the use of subject